src/flex-example-app-ios.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.EnterpriseHeyzap.HeyzapSDKTest</id>
```

Comment:
okay, i will add some instructions on how to change code to get it worked with users accounts